Source/Scene/Scene.js

```diff
@@ -1964,126 +1963,6 @@ Scene.prototype.isVisible = function (command, cullingVolume, occluder) {
   );
 };
 
-function getAttributeLocations(shaderProgram) {
```

Comment:
Great idea to move this into it's own file and make `Scene` a little less cluttered